cpp/src/replace/replace.cu

```diff
@@ -153,14 +159,18 @@ void replace_nulls_with_scalar(gdf_size_type size, Type* out_data, gdf_valid_typ
   int step = blksz * gridsz;
 
   for (int i=start; i<size; i+=step) {
-    out_data[i] = gdf_is_valid(out_valid, i)? out_data[i] : *in_data_scalar;
+    out_data[i] = gdf_is_valid(in_valid, i)? in_data[i] : *replacement;
```

Comment:
We're phasing out `gdf_is_valid` because it _always_ checks if `in_valid == nullptr`.

Instead, use https://github.com/rapidsai/cudf/blob/branch-0.8/cpp/src/bitmask/bit_mask.cuh#L165

You'll need to `reinterpret_cast` `invalid` to `bit_mask_t` before calling the kernel.

You just need to guarantee that `in_valid` isn't `nullptr`. Since this function is specifically for replacing nulls... this shouldn't be difficult.